todolist/index.html

```diff
@@ -20,7 +20,7 @@ <h2>Today</h2>
     </div>
     <div id="input-text">
       <input type="text" name="inputlist" id="textInput" maxlength="32">
-      <button type="submit" id="textSumbit">⏎</button>
+      <button type="submit" id="textSubmit">⏎</button>
```

Comment:
```suggestion
      <button type="submit" id="text-submit">⏎</button>
```

아이디 이름과 클래스 이름은 `hyphen-case`로 짓습니다.